app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -1,31 +1,19 @@
-require 'barby'
-require 'barby/barcode/code_128'
-require 'barby/outputter/png_outputter'
-
 module Idv
   module InPerson
     class ReadyToVerifyPresenter
       # WILLFIX: With LG-6881, confirm timezone or use deadline from enrollment response.
       USPS_SERVER_TIMEZONE = ActiveSupport::TimeZone['America/New_York']
 
-      delegate :selected_location_details, to: :enrollment
+      delegate :selected_location_details, :enrollment_code, :profile, to: :enrollment
 
       def initialize(enrollment:)
         @enrollment = enrollment
       end
 
-      def barcode_data_url
```

Comment:
was this previously a placeholder img until we could get barcodes working?

Reply:
We used to generate an image using the barby gem, but Andrew realized that was incompatible with email, so he switched to using the pure-HTML version of the barcode in both places.

Reply:
ahh ok